src/app/model/Rock.ts

```diff
@@ -0,0 +1,54 @@
+export class Rock implements PieceMovement {
+  id: string;
+  side: string;
+  position: string;
+
+  constructor(id: string, side: string, position: string) {
+    this.id = id;
+    this.side = side;
+    this.position = position;
+  }
+  possibleMoves(position: string, board: Board): string[] {
+    position = this.position;
+    let column = position[0];
+    let row = position[1];
+    let movesToGo: string[] = [];
+    // do sth with movesToGo...
+    if (column === 'A') {
```

Comment:
Dobrze jakbyś napisał do tego test, bo zdaje się prosty w miarę. Skoro sobie założysz gdzie stoi wierza np. i jaka jest szachownica.

Reply:
Tak tak, testy napiszę, tylko najpierw chciałem to skonsultować, bo nie wiedziałem, czy wgl idę dobrym torem :-)